apps/test/unit/templates/curriculumCatalog/CurriculumCatalogTest.jsx

```diff
@@ -28,17 +28,24 @@ import {
 
 describe('CurriculumCatalog', () => {
   const defaultProps = {curriculaData: allCurricula, isEnglish: false};
+  let replaceStateOrig = window.history.replaceState;
 
   beforeEach(() => {
     const store = configureStore({reducer: {responsive}});
     store.dispatch(setResponsiveSize(ResponsiveSize.lg));
+    window.history.replaceState = (_, __, newLocation) => newLocation;
```

Comment:
Trying to parse what's happening here –– could this also be `window.history.replaceState = replaceStateOrig`? Or is there something else going on with this? Looking at https://developer.mozilla.org/en-US/docs/Web/API/History/replaceState for reference

Reply:
Oh shoot thank you for pointing this out! I believe the other two lines are actually useless and I just need this single line to be set once. This is setting the replaceState function to return the string of the newLocation url since that's all that I'm testing.